### ISSUE TITLE
[BUILD] Kotlin script compilation NullPointerException when building with kotlin version 1.4.30

### ISSUE THREAD

**Issue by bakic** [theopenconversationkit/tock#1199]
The build is no more working with a NullPointerException error related to kotlin

```
[ERROR] Failed to execute goal org.jetbrains.kotlin:kotlin-maven-plugin:1.4.30:compile (compile) on project tock-nlp-front-shared: Compilation failure
[ERROR] java.lang.NullPointerException
[ERROR] 	at com.intellij.openapi.util.KeyedExtensionCollector.getPoint(KeyedExtensionCollector.java:203)
[ERROR] 	at com.intellij.openapi.util.KeyedExtensionCollector.getExtensions(KeyedExtensionCollector.java:135)
[ERROR] 	at com.intellij.openapi.util.KeyedExtensionCollector.buildExtensions(KeyedExtensionCollector.java:124)
[ERROR] 	at com.intellij.lang.LanguageExtension.buildExtensions(LanguageExtension.java:157)
[ERROR] 	at com.intellij.lang.LanguageExtension.buildExtensions(LanguageExtension.java:26)
[ERROR] 	at com.intellij.openapi.util.KeyedExtensionCollector.forKey(KeyedExtensionCollector.java:109)
```
It seems that the latest version of kotlin 1.4.30 has some issues with the parallel build.
There are issues open related to that in jetbrains:
https://youtrack.jetbrains.com/issue/KT-45007
https://youtrack.jetbrains.com/issue/KT-44931

Maybe we should go back to the previous version for the moment
@vsct-jburet @francoisno

**Comment by vsct-jburet** [comment]
It is not that easy to downgrade the kotlin version. As it only a random parallel build failure, I suggest to stay like this for now

**Comment by vsct-jburet** [comment]
Il sera fixé en 1.4.32 https://youtrack.jetbrains.com/issue/KT-45007 je fais l'upgrade dès que la version est dispo

**Comment by vsct-jburet** [comment]
En fait il y a même un fix intermédiaire que je vais faire immédiatement
